ambry-tools/src/main/java/com.github.ambry/store/StoreCopier.java

```diff
@@ -178,9 +178,11 @@ public static void main(String[] args) throws Exception {
       File srcDir = new File(config.srcStoreDirPath);
       File tgtDir = new File(config.tgtStoreDirPath);
       StoreMetrics metrics = new StoreMetrics(clusterMap.getMetricRegistry());
+      DiskSpaceAllocator diskSpaceAllocator =
+          new DiskSpaceAllocator(false, null, 0, new StorageManagerMetrics(clusterMap.getMetricRegistry()));
       try (StoreCopier storeCopier = new StoreCopier("src", srcDir, tgtDir, config.storeCapacity,
-          config.fetchSizeInBytes, storeConfig, metrics, storeKeyFactory, new DiskIOScheduler(null),
-          Collections.EMPTY_LIST, SystemTime.getInstance())) {
+          config.fetchSizeInBytes, storeConfig, metrics, storeKeyFactory,
+          new DiskIOScheduler(null),diskSpaceAllocator, Collections.emptyList(), SystemTime.getInstance())) {
```

Comment:
formatting? Looks like there needs to be a space b/w the comma and `diskSpaceAllocator` here.

Reply:
fixed